packages/neon/neon/lib/src/widgets/unified_search_results.dart

```diff
@@ -29,22 +29,31 @@ class NeonUnifiedSearchResults extends StatelessWidget {
     final bloc = accountsBloc.activeUnifiedSearchBloc;
     return ResultBuilder.behaviorSubject(
       stream: bloc.results,
-      builder: (final context, final results) => NeonListView(
-        items: results.data?.entries,
-        isLoading: results.isLoading,
-        error: results.error,
-        onRefresh: bloc.refresh,
-        builder: (final context, final snapshot) => AnimatedSize(
-          duration: const Duration(milliseconds: 100),
-          child: _buildProvider(
-            context,
-            accountsBloc,
-            bloc,
-            snapshot.key,
-            snapshot.value,
-          ),
-        ),
-      ),
+      builder: (final context, final results) {
+        final values = results.data?.entries.toList();
+
+        return NeonListView(
+          isLoading: results.isLoading,
+          error: results.error,
+          onRefresh: bloc.refresh,
+          itemCount: values?.length ?? 0,
+          itemBuilder: (final context, final index) {
+            final snapshot = values![index];
+
+            // TODO: use animated list with NeonListView.custom
```

Comment:
This should probably be a new issue too

Reply:
It's already in #802

Reply:
Then please remove the todo in the code. I think we should never have todos in the code (at least for code that is merged) and directly create issues for them.